change/@ni-nimble-components-d948f8c3-7e90-4ec6-b1f3-468c20ad234d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Am I correct that the styling changes will impact clients by adding additional space around their checkboxes? For example, in the example app it looks like the checkbox has some extra space on all sides so no longer left aligns with the label above it.
https://60e89457a987cf003efc0a5b-ndvckxqkhv.chromatic.com/example-client-app/#/customapp
https://nimble.ni.dev/storybook/example-client-app/#/customapp

Is that an intentional design change? If so then I think we should:
1. mention it in the change file comment
2. spot check a few SLE apps to ensure it looks ok (I think Fred has complained that a lot of slide outs already have bad checkbox layouts so maybe this is a low bar)

I don't think it merits being marked as a breaking change or doing additional notifications on Teams, but we could consider those too if it seems disruptive.

Reply:
The padding is coming from the [visual design specs](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=8729-97569&node-type=frame&t=F4QOnnPRawZhNAV2-0). Perhaps we need to have a discussion with Brandon if we don't agree with the design.

Reply:
I think either way I'd vote to split it out into a separate PR and not make the change here. If we do end up wanting it, I'd argue implementation-wise for the checkbox (which is a standalone control) that the padding / margin would be responsibility of the thing that is laying it out.

The padding is for a grouped, labelled set of checkboxes, not for every possible composition of a checkbox. We'd have a separate PR to capture the layout guidance in storybook docs or create something like a checkbox-group to help layout and we would also want to update radio button group at the same time.

Reply:
I'll pull that change out of this PR.

Reply:
The left padding has now been removed from the checkbox. The square control is back to being flush with the left edge of the `nimble-checkbox`.

Reply:
The Chromatic diffs still show some additional vertical padding. Is that a client-facing change too? If so, what makes us ok with the vertical padding change but not the left padding change?

Reply:
The vertical control size change makes the control height consistent with out other controls. The left padding addition makes the start of the control bounds inconsistent with out other controls (and even if the design is correct, I'm arguing that is not the implementation we should make with the discussion above)

Reply:
The updated change file comment looks good, I think that should give sufficient breadcrumbs if clients see this change and wonder what happened.